mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/Dsl.kt

```diff
@@ -128,19 +130,22 @@ suspend fun ComposeContentTestRule.setContentWithTestEnvironment(
                   dataStoreFactory = dataStoreFactory,
                   store = store,
               ),
-          strings = strings,
+          // This ignores the language in the dataStoreFactory
+          strings = English,
           user = user,
       )
-  setContentWithLocalizedStrings(strings) {
+  setContent {
     PawniesTheme {
       ProvideFacades(
           authentication = authenticationFacade,
           social = socialFacade,
           chess = chessFacade,
           speech = speechFacade,
           tournament = tournamentFacade,
-          content = { with(environment) { content() } },
-      )
+          settings = settingsFacade,
+      ) {
+        ProvideLocalizedStrings(settingsFacade = settingsFacade) { with(environment) { content() } }
```

Comment:
(if you provide a default argument to `ProvideLocalizedStrings`)

```suggestion
        ProvideLocalizedStrings { with(environment) { content() } }
```